pkg/kubectl/describe.go

```diff
@@ -1779,10 +1779,18 @@ func (d *ServiceAccountDescriber) Describe(namespace, name string, describerSett
 		}
 	}
 
-	return describeServiceAccount(serviceAccount, tokens)
+	existingPullSecrets := map[string]api.Secret{}
+	pullSecrets, err := d.Core().Secrets(namespace).List(api.ListOptions{})
+	if err == nil {
```

Comment:
@AdoHe 
> could we narrow this list? any reason to list all secrets on the namespace?

The reason why I did not filter this list was because I was not sure exactly which "types" of secrets could be considered `pullSecrets`. For example, service accounts I create come with a `kubernetes.io/dockercfg` secret type. However, any new secrets I add as a user, are of type `Opaque`.

If these are the only two types that apply to `pullSecrets`, I could filter it to these